pages/sponsors.html

```diff
@@ -8,21 +8,28 @@
+        <p>The PrivacyTools website and services are a community project. There is no advertising, affiliate links, or other forms of monetization.<br><strong>Your donations here directly support hosting this website and compensating contributors to this project.</strong></p>
         <p>
           <a href="https://opencollective.com/privacytoolsio#section-contribute" class="btn btn-success mb-5">Become a Sponsor</a>
-          <a href="/donate/" class="btn btn-outline-danger mb-5"><span class="fas fa-heart fa-fw"></span> Donate Directly</a>
+          <a href="/donate/" class="btn btn-outline-danger mb-5"><span class="fas fa-heart fa-fw"></span> Donate to PrivacyTools</a>
         </p>
       </div>
     </div>
   </div>
-  <h3>Why sponsor {{ site.name }}?</h3>
-  <p>This sponsorship program is designed to allow companies, organizations, and individuals partner with the {{ site.name }} team to support our vision of a more privacy-respecting internet and the greater online community.</p>
+  <h3>Why sponsor PrivacyTools?</h3>
+  <p>This sponsorship program is designed to allow companies, organizations, and individuals partner with the PrivacyTools team to support our vision of a more privacy-respecting internet and the greater online community.</p>
   <p>With this exposure and sponsorship, your customers will recognize your intrinsic understanding and commitment to user privacy. Moreover, you'll directly contribute to our mission of spreading privacy-respecting tools and knowledge worldwide!</p>
-  <p>As a sponsor of {{ site.name }}, your company will be widely recognized in a variety of ways, some of which we've detailed below.</p>
+  <p>At PrivacyTools, we believe in...</p>
+  <ul>
+    <li><strong>Privacy</strong>, of course. Privacy gives you as a user control over how your information is used and spread online. Trustworthy and secure products and services are the key to that future.</li>
+    <li><strong>Security</strong>. We believe services cannot provide privacy at all without being secure. Strong encryption is the only way to prevent malicious actors from snooping in on our communications.</li>
+    <li><strong>Freedom</strong>. The right to express your opinion online without interference or surveillance is the only way we can grow as a society, and privacy-respecting tools should foster that growth.</li>
+  </ul>
+  <p>We reserve the right or deny all sponsors from receiving the benefits detailed below if we believe that your organization or product is not aligned with our key values. Therefore, if you are interested in sponsoring our project, please first email <a href="mailto:sponsors@privacytools.io">sponsors@privacytools.io</a> so we may discuss further.</p>
```

Comment:
```suggestion
  <p>We reserve the right to deny all sponsors from receiving the benefits detailed below if we believe that your organization or product is not aligned with our key values. Therefore, if you are interested in sponsoring our project, please first email <a href="mailto:sponsors@privacytools.io">sponsors@privacytools.io</a> so we may discuss further.</p>
```

Right or deny makes no sense to me, did you perhaps mean right to?